WooCommerce/Classes/System/WooConstants.swift

```diff
@@ -13,6 +13,10 @@ public enum WooConstants {
     ///
     static let databaseStackName = "WooCommerce"
 
+    /// Local SQLite Database Name
+    ///
+    static let localSQLiteDatabaseName = "woo-local.db"
```

Comment:
nit: any reasons not to use the `*.sqlite` extension as in the [example in the doc](https://swiftpackageindex.com/groue/grdb.swift/v7.6.1/documentation/grdb#Usage)? might we want to name it like `WooCommerce-local` just to be more consistent with the existing sqlite files from Core Data?

I was still able to open the database in DataGrip with `woo-local.db` in the directory, so just a non-blocking question.

Reply:
No, I'd put more thought into the name than the extension. `.sqlite` is more descriptive, let's use that.

We could go to `WooCommerce-local` – I didn't, in order to put a bit more space between the Core Data name and the GRDB name. But I'm not holding that view super strongly. I'll leave it as it is for now, but if you feel strongly just let me know and I'll change it 😊